Some LaTeX installations have only utf8, not utf8x

utf8x adds support for much more Unicode characters, but these are not
needed to typeset the mathematics in the Sage documentation

--- Sphinx-1.2.2/sphinx/ext/pngmath.py.orig	2014-03-02 20:38:09.000000000 +1300
+++ Sphinx-1.2.2/sphinx/ext/pngmath.py	2014-10-19 23:35:17.000000000 +1300
@@ -42,7 +42,7 @@
 
 DOC_HEAD = r'''
 \documentclass[12pt]{article}
-\usepackage[utf8x]{inputenc}
+\usepackage[utf8]{inputenc}
 \usepackage{amsmath}
 \usepackage{amsthm}
 \usepackage{amssymb}
